src/syzygy/tbprobe.cpp

```diff
@@ -186,7 +192,13 @@ const char* PawnlessDtzSuffixes[VARIANT_NB] = {
     nullptr,
 #endif
 #ifdef THREECHECK
-    nullptr
+    nullptr,
+#endif
+#ifdef SUICIDE
+    ".stbz",
```

Comment:
I am confused. Aren't pawnless suicide tables used for giveaway and not the other way around?